roles/confluent.test/molecule/rbac-kerberos-debian/molecule.yml

```diff
@@ -209,6 +209,10 @@ provisioner:
           ldap.group.member.attribute.pattern=CN=(.*),OU=rbac,DC=example,DC=com
           ldap.user.object.class=account
 
+      ksql:
```

Comment:
does this test validate ksql processing logs? `ksql_log_streaming_enabled` defaults to false, i think you need to turn it on and add verify tasks

Reply:
When we had discussed this previously, you stated that you were fine with me implementing this and validating that it works.  I can write additional test, but that wasn't what was previously discussed thus why its not here.

Reply:
If you want to go the route of not adding any tests for this feature then i dont think there needs to be any changes to this file

but bc there is such complex rbac/kerberos code it'd probably be best to add a test